chia/full_node/coin_store.py

```diff
@@ -154,7 +152,7 @@ async def get_coin_records_by_puzzle_hash(
         coins = set()
         cursor = await self.coin_record_db.execute(
             f"SELECT * from coin_record WHERE puzzle_hash=? AND confirmed_index>=? AND confirmed_index<? "
-            f"{'' if include_spent_coins else 'AND spent=0'}",
+            f"{'' if include_spent_coins else 'AND +spent=0'}",
```

Comment:
I'd prefer using INDEXED BY rather than this + trick as it's more readable.
SELECT * from coin_record INDEXED BY coin_puzzle_hash WHERE...